build.sh

```diff
@@ -1,9 +1,87 @@
 #!/usr/bin/env bash
+set -e	# Stop on error
```

Comment:
No line feed at the end of file. Also this needs to be ShellCheck'ed ideally

Reply:
Passes shellcheck, will fix the whitespace.

Reply:
Fixed.